homeassistant/components/media_player/intent.py

```diff
@@ -41,10 +43,88 @@ async def async_setup_intents(hass: HomeAssistant) -> None:
             INTENT_SET_VOLUME,
             DOMAIN,
             SERVICE_VOLUME_SET,
-            extra_slots={
+            required_domains={DOMAIN},
+            required_states={MediaPlayerState.PLAYING},
```

Comment:
this is preferred, but is this required?

Reply:
It's required in the "multiple targets" case (according to the spec, at least). I didn't differentiate between the single and multiple target cases in this PR, and I'm not convinced we should in general. It's now always clear when there will be a single target, so I went with a more conservative approach to start.